.github/workflows/weekly_tests.yaml

```diff
@@ -14,94 +14,45 @@ jobs:
+          - charm-logrotated
+          - charm-nginx
+          - charm-nrpe
+          - charm-openstack-service-checks
+          - charm-prometheus-blackbox-exporter
+          - charm-prometheus-juju-exporter
+          - charm-prometheus-libvirt-exporter
+          - charm-simple-streams
+          - charm-storage-connector
+          - charm-sysconfig
+          - charm-userdir-ldap
+          - charmed-openstack-upgrader
+          - juju-backup-all
+          - juju-lint
+          - openstack-exporter-operator
+          - prometheus-hardware-exporter
+          - prometheus-juju-backup-all-exporter
+        workflow_file_name:
+          - check.yaml
         include:
-          - repo: charm-advanced-routing
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-apt-mirror
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-cloudsupport
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-duplicity
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-juju-backup-all
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-juju-local
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-kubernetes-service-checks
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-local-users
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-logrotated
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-nginx
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-nrpe
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-openstack-service-checks
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-prometheus-blackbox-exporter
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-prometheus-juju-exporter
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-prometheus-libvirt-exporter
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-simple-streams
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-storage-connector
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-sysconfig
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charm-userdir-ldap
-            workflow_file_name: check.yaml
-            branch: main
-          - repo: charmed-openstack-upgrader
-            workflow_file_name: check.yaml
-            branch: main
           - repo: hardware-observer-operator
             workflow_file_name: check.yaml
```

Comment:
did you forget to include this in `matrix`?

Reply:
No I put it here to be clearer, because it's there twice (once for check.yaml, once for cos_integration.yaml).

Reply:
isn't both `check.yaml`?

Reply:
oh, sorry. I must be blind...